conf/deploydb.sh

```
@@ -42,6 +42,7 @@ mkdir -p $flyway_sql
 
 cp $base/db/V0.6__schema.sql $flyway_sql
 cp $base/db/upgrade/* $flyway_sql
+cp $base/db/zsv/* $flyway_sql
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)